docs/source/scalarstats.rst

```diff
@@ -158,6 +158,10 @@ Quantile and Friends
     w = rand(n)
     xk = median(x, weights(w))
 
+.. function:: wquantile(x, w, q)
+
+  Compute the weighted quantile of ``x``, using weights given by a weight vector ``w`` (of type ``WeightVec``).  Weights must not be negative. The weight and data vectors must have the same length `n`. Quantiles as defined as follow: assume for simplicity that weights sum to `n`. Denote :math:`x_h` the smallest element of ``x`` such that the cumulated sum up of weights up to `x_h` is strictly superior to :math:`lv * q + m`. Denote :math:`x_l` the element just before `x_h`. The sample quantile is defined as `(1-gamma) x_l + gamma x_h`. 
```

Comment:
Oh, and you say "quantile" (singular) while a vector of probabilities is supported. It would make sense to use the plural, and to say a word about the probabilities, following the vocabulary used for `?quantile`.